packages/web-app-files/src/components/SideBar/Versions/FileVersions.vue

```diff
@@ -3,20 +3,25 @@
     <oc-loader v-if="loading" />
     <oc-table-simple v-if="!loading && hasVersion">
       <oc-tbody>
-        <oc-tr v-for="(item, index) in versions" :key="index" class="file-row">
-          <oc-td width="shrink">
+        <oc-tr v-for="(item, index) in versions" :key="index" data-testid="file-row">
```

Comment:
This test id is way to generic. Please make sure to be specific in naming them, e.g. using `data-testid="file-version-table-row"` (also goes for the other `data-testid`s in this test. Perhaps they crash CI because of naming conflicts right now)

Reply:
I thought the selector was added by me and refactored it to `data-testid`. 
reverted this one.
suggested changes are made for other occurrences